tests/reference/asr-unsigned_02-6563e58.stderr

```diff
@@ -0,0 +1,5 @@
+semantic error: The result of the unary minus `-` operation is negative, thus out of range for u16
+ --> tests/errors/unsigned_02.py:3:10
+  |
+3 | i: u16 = -u16(5)
+  |          ^^^^^^^ use -i32(u) for signed result
```

Comment:
It would be great if it could copy the actual expression inside:
```suggestion
  |          ^^^^^^^ use -i32(5) for signed result
```
But I guess for that we would have to have an ASR -> Python printer, which we plan to, but right now. So we can do that later.